tabular/src/autogluon/tabular/configs/presets_configs.py

```diff
@@ -21,10 +21,10 @@
         "time_limit": 3600,
         "refit_full": True,
         "set_best_to_refit_full": True,
-        "_save_bag_folds": False,
+        # "_save_bag_folds": False,
```

Comment:
Is this just a temporary change? I see there is a hack related to this option.

Reply:
Temporary change to check performance.

Reply:
Change has been reverted after testing